src/plugin/regions/region.js

```diff
@@ -611,4 +612,15 @@ export class Region {
             });
         }
     }
+
+    updateHandlesResize(resize) {
+        const cursorStyle = resize ? 'col-resize' : 'none';
+        const childNodes = this.element.childNodes;
+
+        childNodes.length > 0 && childNodes.forEach(child => {
```

Comment:
IMO it'd be better to store the handle elements when we create them  in `render()`.  Checking `nodeName` here feels gross.

Reply:
Yea, thanks for suggestion